package.json

```diff
@@ -2,22 +2,20 @@
   "name": "spectacle",
   "version": "5.5.0",
   "description": "ReactJS Powered Presentation Framework",
+  "source": "src/components/index.js",
   "main": "lib/index.js",
   "module": "es/index.js",
   "types": "index.d.ts",
   "scripts": {
-    "build-wds": "webpack-dev-server --config webpack.config.example.js",
-    "build-webpack-base": "webpack",
-    "build-webpack-example": "builder run build-webpack-base -- --config webpack.config.example.js",
-    "build-webpack-lib-dev": "builder run build-webpack-base -- --config webpack.config.dev.js",
-    "build-webpack-lib": "builder run build-webpack-base",
-    "build": "builder concurrent --buffer build-webpack-lib build-webpack-lib-dev build-webpack-example",
     "lint": "eslint --ignore-path .gitignore .",
     "lint-fix": "yarn lint --fix",
     "prettier": "prettier \"**/*.{js,json,ts,css,md}\"",
-    "prettier-fix": "yarn prettier --write",
```

Comment:
We use `yarn` for this project so let's standardize on that instead of `npm` for shelling commands.